docker-compose.yml

```diff
@@ -1,17 +1,31 @@
-web:
-  build: .
-  dockerfile: Dockerfile
-  ports:
-    - 8000:8000
-  links:
-    - redis
-  environment:
-    - PYTHONUNBUFFERED=1
-    - PYTHONASYNCIODEBUG=1
-    - GENE_TO_REACTIONS_API
-    - GENE_TO_REACTIONS_PORT
-    - ID_MAPPER_API
-redis:
-  image: redis
-  ports:
-    - 6379
+version: "2.1"
+services:
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    ports:
+      - 8000:8000
+    links:
+      - redis
+    environment:
+      - PYTHONUNBUFFERED=1
+      - PYTHONASYNCIODEBUG=1
+      - GENE_TO_REACTIONS_API=${GENE_TO_REACTIONS_API:-139.59.134.152}
+      - GENE_TO_REACTIONS_PORT=${GENE_TO_REACTIONS_PORT:-50053}
+      - ID_MAPPER_API=${ID_MAPPER_API:-https://api.dd-decaf.eu/idmapping/query}
+      - REDIS_ADDR=${REDIS_ADDR:-redis}
+      - REDIS_PORT=${REDIS_PORT:-6379}
+    networks:
+      - default
+      - iloop-net
```

Comment:
Might want to give this service an alias on the network since we pretty much name every outward facing service "web".